azurerm/resource_arm_sql_administrator.go

```diff
@@ -0,0 +1,143 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/sql/mgmt/2015-05-01-preview/sql"
+	"github.com/hashicorp/terraform/helper/schema"
+	uuid "github.com/satori/go.uuid"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	//"github.com/hashicorp/terraform/helper/validation"
+	//"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+)
+
+func resourceArmSqlAdministrator() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmSqlAdministratorCreateUpdate,
+		Read:   resourceArmSqlAdministratorRead,
+		Update: resourceArmSqlAdministratorCreateUpdate,
+		Delete: resourceArmSqlAdministratorDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"server_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"resource_group_name": resourceGroupNameSchema(),
+
+			"login": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"object_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"tenant_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceArmSqlAdministratorCreateUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).sqlServerAzureADAdministratorsClient
+	ctx := meta.(*ArmClient).StopContext
+
+	serverName := d.Get("server_name").(string)
+	resGroup := d.Get("resource_group_name").(string)
+	administratorName := "activeDirectory"
```

Comment:
_note_ when #825 is resolved we shouldn't need this field

Reply:
presuming I don't need to do anything here?